targets/TARGET_NUVOTON/TARGET_M480/crypto/crypto-misc.c

```diff
@@ -102,6 +110,15 @@ void crypto_zeroize(void *v, size_t n)
     }
 }
 
+/* Implementation that should never be optimized out by the compiler */
+void crypto_zeroize32(uint32_t *v, size_t n)
+{
+    volatile uint32_t *p = (uint32_t*) v;
```

Comment:
This should probably be `(uint32_t volatile *) v`.

Reply:
@hanno-arm I fixed it in https://github.com/ARMmbed/mbed-os/pull/5812/commits/cfdc72d75ef9bdfddcb9d7ae35ad2a41f191bb93.

Reply:
Thanks!